servicetalk-gradle-plugin-internal/src/main/groovy/io/servicetalk/gradle/plugin/internal/ServiceTalkLibraryPlugin.groovy

```diff
@@ -56,8 +57,16 @@ final class ServiceTalkLibraryPlugin extends ServiceTalkCorePlugin {
     project.configure(project) {
       pluginManager.apply("java-library")
 
-      sourceCompatibility = TARGET_VERSION
-      targetCompatibility = TARGET_VERSION
```

Comment:
Same questions here. I'm hesitant to remove these properties. Your change doesn't look conflicting.